features/profiler.yml

```diff
@@ -1,7 +1,7 @@
-draft_date: 2024-08-19
-name: JS Self-Profiling API
-description: TODO
+name: Profiler
+description: The `Profiler` API traces and records JavaScript execution on a page, which you can send to a server for later analysis.
 spec: https://wicg.github.io/js-self-profiling/
+group: javascript
```

Comment:
So far, everything in `javascript` has been a language feature (e.g., stuff you'd find in V8). But this is a Web API. So I'm not 100% sure about this grouping. Maybe @Elchi3 has an opinion on this?

```suggestion
```

Reply:
Yeah I think it would be good to make this group about ECMAScript features for now only. I think this API would be a better fit in a group called "performance"? 

Even if the spec talks about measuring JS execution, it really measures execution of any scripts on a page, not just ECMAScript language performance, I think.

This brings me back to a point that I made to Daniel last week: A lot of the web-feature IDs and groups are informed by how a specification has been named. But what are the guidelines for spec authors for how to name a spec? Maybe we can feedback into spec authoring to ensure there is more consistency and less confusing usage of "XYZ API" vs "WebXYZ API" vs "JS XYZ API", etc.

Reply:
Helpful comment, Florian. Thank you! I made another suggestion on the description to go with this group removal, then.

Also, I've put on the agenda for today's Baseline/web-features call to see if anyone knows about how to contribute naming guidance to spec authors. It's a really good idea!

Reply:
W3C has a [manual of style](https://www.w3.org/2001/06/manual/), but it has received very limited maintenance and its [current guidance on Title](https://www.w3.org/2001/06/manual/#Title) doesn't help with the specific aspect you mention.

Discussing better titling/naming conventions based on the  experience on Web features would be great; I think the [spec-prod](https://lists.w3.org/Archives/Public/spec-prod/) mailing list would likely be the right place to start such a discussion (and it could be informed by extracting stats from browser-specs / web-features).